internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixReadOnlySchemaRepository.java

```diff
@@ -553,7 +432,6 @@ private void populateSchemaMap(String storeName) {
       accessor.getAllValueSchemas(storeName).forEach(schemaData::addValueSchema);
 
       // Fetch derived schemas if they are existing
-      Store store = storeRepository.getStoreOrThrow(storeName);
```

Comment:
Is it the caller's responsibility to ensure that `store` exists?

Reply:
Yes, that was my intent.